sendmail.go

```diff
@@ -87,19 +103,41 @@ func (m *Mail) Send() error {
 	if len(out) != 0 {
 		return errors.New(string(out))
 	}
-	return sendmail.Wait()
+	return cmd.Wait()
 }
 
 // WriteTo writes headers and content of the email to io.Writer
-func (m *Mail) WriteTo(wr io.Writer) error {
+func (m *Mail) WriteTo(wr io.Writer) error { // nolint: vet
```

Comment:
If we are breaking compatibility with previous versions, then return values from this function can be also changed.